Debian-specific patch: add /var/lib/doc-base to the list of dirs that 
librarian will parse.

Index: yelp-2.22.1/src/yelp-main.c
===================================================================
--- yelp-2.22.1.orig/src/yelp-main.c	2008-05-18 17:57:16.446670073 +0200
+++ yelp-2.22.1/src/yelp-main.c	2008-05-18 18:10:37.918680996 +0200
@@ -341,6 +341,7 @@ main (int argc, char **argv) 
 	GnomeClient   *client;
 	gboolean       session_started = FALSE;
 	gchar *local_id;
+	gchar *xdg_dirs;
 	GOptionContext *context;
 
 	bindtextdomain(GETTEXT_PACKAGE, GNOMELOCALEDIR);  
@@ -354,6 +355,15 @@ main (int argc, char **argv) 
 		putenv ("DESKTOP_STARTUP_ID=");
 	}
 
+	xdg_dirs = (gchar *) g_getenv ("XDG_DATA_DIRS");
+	if (xdg_dirs) {
+		xdg_dirs = g_strconcat (xdg_dirs, ":/var/lib/doc-base", NULL);
+		g_setenv ("XDG_DATA_DIRS", xdg_dirs, TRUE);
+		g_free (xdg_dirs);
+	} else {
+		g_setenv ("XDG_DATA_DIRS", "/usr/local/share:/usr/share:/var/lib/doc-base", TRUE);
+	}
+
 	/* Commandline parsing is done here */
 	context = g_option_context_new (N_(" GNOME Help Browser"));
 
